htdocs/core/lib/functions.lib.php

```diff
@@ -71,6 +71,27 @@ function utf8_decode($elements)
 	}
 }
 
+/**
+ * Return multidir output for an Dolibarr object
+ *
+ * @param CommonObject $object Dolibarr common object
+ * @param string $module override object element, by example mycompany instead societe
+ * @since Dolibarr V18
+ * @return string|void
+ */
+function getMultidirOutput($object, $module = '')
+{
+	global $conf;
+	if (!is_object($object)) {
+		return;
```

Comment:
The function is used by doing 
$var = function()
So we should not have void as return. May be we can return null or '' ?